client/app/components/Modal/index.jsx

```diff
@@ -97,23 +98,53 @@ export class Modal extends React.Component<Props, State> {
     this.setState({ open: !open });
   };
 
-  render() {
+  resolveElement = () => {
     const { element, elementId } = this.props;
+    let renderComponent;
+
+    if (typeof element === 'object' && element.component) {
+      const { component, props } = element;
+      renderComponent = React.createElement(
+        this.resolveComponent(component),
+        { ...props },
+        null,
```

Comment:
Why is null being passed?

Reply:
I'm actually brand new to the `createElement` API and figured I _had_ to pass a value to the `children` argument, even if I didn't explicitly want children.

Reply:
Aha, thanks for explaining. I think you can actually exclude that third argument entirely (it appears to be an optional value). I did that for client/app/mocks/InputMocks.jsx and it worked :P